### ISSUE TITLE
Commercial customers would get value from having more information for each DA

### ISSUE THREAD

**Issue by jamezpolley** [openaustralia/planningalerts#1357]
We've had many requests from potential commercial customers who would like to have more of the details of each DA captured.

For instance, many councils list an "estimated cost"; many differentiate between commercial and residential properties; many differentiate between minor extensions and complete redevelopment, and so on.

At present, our data model only captures a very minimal set of data. This is useful as it's a set of data we can catch consistently across most authorities; any attempt to go beyond this is going to make scraping more complicated.

However, anything we can do to capture more data would be useful.

#1294 would become easier if we had more data to work with for creating tags

We would probably want to implement #1317 first, so that we're able to update this data as and when updates are made on the council site.

**Comment by jamezpolley** [comment]
The fields in the ATDIS [info section](https://www.planningalerts.org.au/atdis/specification#section4.3.1) are probably a good starting place

**Comment by mlandauer** [comment]
Related (on a purely technical level of it adding new fields to the data model) - see #40

**Comment by jamezpolley** [comment]
Having looked into this in a bit more detail, I'm not sure that there's much more information we can provide.

Specifically, the things I've looked at are:

- Estimated cost
  - Not available for all authorities
  - Simplest implementation would be to start adding it where we see
    it
  - It would be nice to have a way to distinguish between "We've
    looked at this scraper and the Authority doesn't provide estimated
    cost" vs "This scraper doesn't have estimated cost just because we
    haven't done anything with it lately". But that would be extra
    complexity for little gain (for us, at least)"
  - Simply making it available for councils that provide it isn't a
    huge amount of work, but doing a pass over all the scrapers would
    be.
- Type
  - ATDIS has both "Application type" and "Development type". 
  - Application Type is not very useful: the values are only used at
    present in NSW, and "DA" is most likely going to cover the bulk of
    the applications.
  - Development type isn't neccessarily very useful either: eg, (this application)[http://epathway.wollongong.nsw.gov.au/ePathway/Production/Web/GeneralEnquiry/EnquiryDetailView.aspx?Id=1077320] simply lists the Development Type as `Development Application`
  - In some cases there *is* more information available; for example,
    (here)[https://pdonline.frasercoast.qld.gov.au/Modules/ApplicationMaster/default.aspx?page=wrapper&key=686198#]. In
    this case we can see in three different ways that this application
    is "Reconfiguring A Lot": that's listed as a `category`; it's in
    the `Details`; and the application reference is `RAL19/0018' -
    seems to mean it's the 18th RAL lodged in 2019.  
  - However... in this case at least, the information is in the
    description we (already
    provide)[https://www.planningalerts.org.au/applications/1163943]
    so I don't think we'd be adding much extra value by looking to
    pull in anything more.
- Description: I haven't made a comprehensive search, but the summary
  we already pull in is actually the most useful detail I've seen on
  the sites I've looked at. To get anything more I think we'd have to
  start scraping details out of attachments on the DAs, where those
  are provided...

In short, it looks like adding a field for the estimated cost is
probably not too difficult, but may not provide a lot of useful
information.

Type doesn't seem to be something we could consistently pull in, and
even where we could, it's probably not going to provide too much extra
value.

Description - we're already pulling in as much detail as we can.

It's feeling to me as though there's not a lot that can be done here
without trying to get a comprehensive sample from every Authority and
trying to manually come up with something like a consistent set of
categorizations across the entire dataset and then finding a way to
mash the data into shape for every scraper.
